OCD-3844: Add ICS column to RWT Report and add ICS as part of the reason for errors or warnings in the report.

kekey1 opened the pull request:
[#OCD-3844]

kekey1 committed:
feat!: Add ICS column to RWT Report and add ICS as part of the reason
for errors or warnings in the report.

[#OCD-3844]

andlar commented on chpl/chpl-resources/src/main/resources/errors.properties:
```diff
@@ -881,10 +881,10 @@ realWorldTesting.upload.listingNotFound=Listing could not be found based on the
-realWorldTesting.report.missingPlansError:ERROR: Listings eligible for Real World Testing in %s must have a Plans URL by %s.
-realWorldTesting.report.missingResultsWarning:WARNING: Listings eligible for Real World Testing in %s must have a Results URL by %s.
-realWorldTesting.report.missingResultsError:ERROR: Listings eligible for Real World Testing in %s must have a Results URL by %s.
+realWorldTesting.report.missingPlansWarning:WARNING: Listing is eligible for Real World Testing in %s%s and must have a Plans URL by %s.
```
Why did the `%s` become `%s%s`?

kekey1 replied:
So that it could say "Listing is eligible for Real World Testing in 2022 due to ICS and must have... " OR "Listing is eligible for Real World Testing in 2022 and must have" (second %s is the "due to ICS" part).

kekey1 committed:
fix: Write RWT Eligibility Report job logs to correct logger

[#OCD-3844]

kekey1 committed:
feat: Add RWT Eligibility check - if a listing has ICS the parent must
be Withdrawn by Developer to consider the listing as RWT Eligible
because of ICS

[#OCD-3844]

kekey1 committed:
fix: Update error message to use 'Inherited Certification Status'
instead of 'ICS'

[#OCD-3844]

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
Merge branch 'staging' into OCD-3844

kekey1 committed:
feat: Update errors/warnings for RWT Report related to ICS

[#OCD-3844]
